tests/migration_test.go

```diff
@@ -1126,6 +1124,13 @@ var _ = Describe("[rfe_id:393][crit:high[vendor:cnv-qe@redhat.com][level:system]
 
 			tests.CreateConfigMap(configMapName, config_data)
 			tests.CreateSecret(secretName, secret_data)
+		}, 60)
```

Comment:
the second argument (`60`) can be dropped as the provided function is called synchronously

Reply:
@vladikr could you address that?

Reply:
Sure, done